lib/std/special/compiler_rt/floatfmodl.zig

```diff
@@ -0,0 +1,129 @@
+const builtin = @import("builtin");
+const std = @import("std");
+
+// fmodl - floating modulo large, returns the remainder of division for f128 types
+// Logic and flow heavily inspired by MUSL fmodl for 113 mantissa digits
+pub fn fmodl(a: f128, b: f128) callconv(.C) f128 {
+    @setRuntimeSafety(false);
```

Comment:
You dont have a big amount of tests, so why not `@setRuntimeSafety(builtin.is_test);` ?

Reply:
This function relay's on integer overflows, enabling it causes the ubsan to trap.

Reply:
Zig has explicit integer overflow (called [wrapping overflow](https://ziglang.org/documentation/master/#Wrapping-Operations)) with `+%`, `-%`, `*%`. Why does this not work?
Silent overflow can lead to overwrite of other register values, which in turn leads to incorrect results.

Reply:
Done in 201761b6